e2e/interchaintestv8/chainconfig/kurtosis.go

```diff
@@ -0,0 +1,199 @@
+package chainconfig
+
+import (
+	"context"
+	"crypto/ecdsa"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/kurtosis-tech/kurtosis/api/golang/core/lib/enclaves"
+	"github.com/kurtosis-tech/kurtosis/api/golang/core/lib/services"
+	"github.com/kurtosis-tech/kurtosis/api/golang/core/lib/starlark_run_config"
+	"github.com/kurtosis-tech/kurtosis/api/golang/engine/lib/kurtosis_context"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/crypto"
+
+	"github.com/strangelove-ventures/interchaintest/v8/testutil"
+
+	"github.com/srdtrk/solidity-ibc-eureka/e2e/v8/ethereum"
+)
+
+const FaucetPrivateKey = "0x4b9f63ecf84210c5366c66d68fa1f5da1fa4f634fad6dfc86178e4d79ff9e59"
```

Comment:
It would be nice if we didn't have this global variable and Faucet private key was randomly generated during setup. We should not use global variable if we can. (These cleanups can be made in a following PR).

If we wanna use a global variable, is it the same address as the anvil chain's faucet?

Reply:
Just like anvil has some predefined address, this one does too. But maybe there is a way to configure it so it uses the same one :P